src/components/core/Button/TextButton.tsx

```diff
@@ -28,7 +28,7 @@ function TextButton({
       onClick={onClick}
       underlineOnHover={underlineOnHover}
       disabled={disabled}
-      data-testId={dataTestId}
+      data-testid={dataTestId}
```

Comment:
this typo was causing errors in the console. everywhere else we're using lowercased `data-testid`